src/App.jsx

```diff
@@ -14,70 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setValue] = useState(goods.find(good => good === 'Jam'));
```

Comment:
```suggestion
  const [selectedGood, setValue] = useState('Jam');
```
anyway there is only strings and you use hardcoded `Jam` to find it